restart_plugin/mtcp_restart_plugin.c

```diff
@@ -849,6 +849,11 @@ mtcp_plugin_skip_memory_region_munmap(Area *area, RestoreInfo *rinfo)
   LhCoreRegions_t *lh_regions_list = NULL;
   int total_lh_regions = lh_info->numCoreRegions;
 
+  // Don't skip munmap of mtcp_restart regions.
```

Comment:
Let's change the comment to remove the double negative:
`// Do an munmap of mtcp_restart regions during restart.  Don't skip this.`

Also, please add a comment about why we need to munmap the mtcp_restart regions within MANA, but we don't need to do that within ordinary DMTCP.  Where is the potential address conflict that we're trying to avoid?

Reply:
I want to add that we should not skip all [heap] but only the [heap] right after the mtcp_restart region.